code/__DEFINES/colors.dm

```diff
@@ -1,7 +1,7 @@
 // This is eventually for wjohn to add more color standardization stuff like I keep asking him >:(
 
 #define COLOR_INPUT_DISABLED "#F0F0F0"
-#define COLOR_INPUT_ENABLED "#D3B5B5"
+#define COLOR_INPUT_ENABLED "#B5D3BD"
```

Comment:
Is it possible for you to make this a preference? I'll kneejerk all over that I don't want to lose pinky grey but realistically I'd rather pick between and then whinge that I don't have the option for white text on a dark background.

Reply:
It's possible, but I'm not sure if it's worth making 2 new prefs for chatbar font color and background color

Reply:
yeah well the new one looks bad to some people